ToolBox/SmartInsider/SmartInsiderConverter.cs

```diff
@@ -255,7 +255,7 @@ private void WriteToFile<T>(DirectoryInfo destinationDirectory, Dictionary<strin
                 fileContents.AddRange(kvp.Value);
 
                 var tsvContents = fileContents
-                    .OrderBy(x => x.LastUpdate)
+                    .OrderBy(x => x.TimeProcessedUtc.Value)
                     .Select(x => x.ToLine())
                     .Distinct();
```

Comment:
The `.Distinct();` call here was the underlying issue since he was dropping data points, is it needed?

Reply:
`.Distinct()` only makes sure we don't have any duplicate lines of CSV, it doesn't affect the number of points a `History` call would get back